### ISSUE TITLE
Format Date field on report

### ISSUE THREAD

**Issue by ScottB-Ronco** [hougaard/Simple-Object-Designer#357]
I created a date field on our Purchase Order screen and made it a field type of DATE.  I also added this field to our Purchase Order report for printing the PO.

However when it prints the date value on the report it shows as date and time like this 5/28/2024 12:00:00 AM.

How can I format the date field on the report so that it shows as May 28, 2024?

**Comment by hougaard** [comment]
Easiest way is to create a calculated field where you format the value with the Format() command...

**Comment by ScottB-Ronco** [comment]
Do you mean adding the already created date field as a calculated field using the Report Calculated Fields option?

If so how do I use the format option to accomplish this?

**Comment by ScottB-Ronco** [comment]
I don't understand what you mean by if dataitem is called header?

Do you mean this is where the date field I created exists? Meaning I created and added it to the purchase header?

**Comment by ScottB-Ronco** [comment]
So I followed your example and created the report calculated field as follows. I created the field on a custom word template report.

![image](https://github.com/hougaard/Simple-Object-Designer/assets/87865315/eb9e8977-3ab4-4aa5-9060-b21665c1a66a)

However after I published the changes I went into custom reports in BC and did update report layout and then exported the report, but I cannot see the field added to the report dataset in the XML Mapping pane.
